src/xunit.performance.metrics/InstructionsRetiredMetricDiscoverer.cs

```diff
@@ -11,7 +11,7 @@ namespace Microsoft.Xunit.Performance
 {
     internal partial class InstructionsRetiredMetricDiscoverer : IPerformanceMetricDiscoverer
     {
-        public const int DefaultInterval = 100000; // Instructions per event.
+        public const int DefaultInterval = 1000000; // Instructions per event.
```

Comment:
I'm thinking that we should probably expose this so that it's clear that if results change by orders of magnitude, it might be this.

Reply:
Agree, but more cleanup needs to happen in order the expose this in a clean manner. The definitions of the Pmc types are in two different assemblies. I would prefer to do this in the future.

---
In reply to: [143036788](https://github.com/Microsoft/xunit-performance/pull/241#discussion_r143036788) [](ancestors = 143036788)